packages/react-dom/index.classic.fb.js

```diff
@@ -33,6 +33,8 @@ export {
   unstable_isNewReconciler,
   unstable_renderSubtreeIntoContainer,
   unstable_runWithPriority, // DO NOT USE: Temporarily exposed to migrate off of Scheduler.runWithPriority.
+  preinit,
+  preload,
```

Comment:
This is making me realize the ReactDOMDispatcher is assumed to have float methods and those methods are not currently feature flagged. Since you turned the flag on everywhere things are going to work but 

1. are we confident the float and singleton changes can go into FB builds?
2. should we flag these functions and make them noops or error if the flag is off?

Reply:
I think we can try it for FB builds and revert if not. It's unlikely to affect anything since FB doesn't use anything stylesheet or head related in React. It's externally managed.